ERROR - 2023-03-01 10:09:59 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:09:59 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 10:11:15 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:11:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 10:13:28 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:13:28 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 10:14:19 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:14:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 10:16:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:16:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 10:16:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:16:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 10:17:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:17:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 10:21:55 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', 'root', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 10:21:55 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 11:31:19 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'webprogmi212', 3306, '', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#7 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-01 11:31:19 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `dmolarte2_news`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 11:31:29 --> mysqli_sql_exception: Table 'webprogmi212.dmolarte2_news' doesn't exist in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-01 11:31:29 --> Table 'webprogmi212.dmolarte2_news' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `dmolarte2_news`', [], false)
 2 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 11:31:31 --> mysqli_sql_exception: Table 'webprogmi212.dmolarte2_news' doesn't exist in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-01 11:31:31 --> Table 'webprogmi212.dmolarte2_news' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `dmolarte2_news`', [], false)
 2 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 11:32:45 --> mysqli_sql_exception: Table 'webprogmi212.dmolarte2_news' doesn't exist in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-01 11:32:45 --> Table 'webprogmi212.dmolarte2_news' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `dmolarte2_news`', [], false)
 2 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-01 11:32:49 --> mysqli_sql_exception: Table 'webprogmi212.dmolarte2_news' doesn't exist in C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\lab3\ci4\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\lab3\ci4\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\lab3\ci4\app\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\lab3\ci4\app\Controllers\News.php(13): App\Models\NewsModel->getNews()
#8 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\News->index()
#9 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#10 C:\xampp\htdocs\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-01 11:32:49 --> Table 'webprogmi212.dmolarte2_news' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `dmolarte2_news`', [], false)
 2 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\NewsModel.php(14): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\News.php(13): App\Models\NewsModel->getNews()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\News->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
